docker-compose.yml

```diff
@@ -100,8 +109,15 @@ x-infrahub-config: &infrahub_config
   INFRAHUB_TRACE_EXPORTER_PROTOCOL: ${INFRAHUB_TRACE_EXPORTER_PROTOCOL:-grpc}
   INFRAHUB_TRACE_EXPORTER_TYPE: ${INFRAHUB_TRACE_EXPORTER_TYPE:-console}
   INFRAHUB_TRACE_INSECURE: ${INFRAHUB_TRACE_INSECURE:-true}
+  INFRAHUB_WORKFLOW_ADDRESS: ${INFRAHUB_WORKFLOW_ADDRESS:-localhost}
+  INFRAHUB_WORKFLOW_DRIVER: ${INFRAHUB_WORKFLOW_DRIVER:-worker}
+  INFRAHUB_WORKFLOW_ENABLE: ${INFRAHUB_WORKFLOW_ENABLE:-true}
```

Comment:
Not sure if we want these settings in there ... not a blocker for this PR but  I think we should find a way to hide these internal settings to the end user

Reply:
I would say, it's the same as some other settings, we allow users to change them and we give them the "default" value.

Reply:
I get that but this list is way too long , it's hard to find the settings that really matter to the end user.